app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -10,10 +10,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     afterRender () {
-      const flowId = this.flow.get('flowId');
-      const flowBegin = this.flow.get('flowBegin');
-
-      this.metrics.logFlowBegin(flowId, flowBegin);
+      this.logFlowEventOnce('begin');
```

Comment:
@philbooth Not really in the scope of this PR, but was trying to figure out the motivation for having `flow-begin-mixin` and if it was still relevant or if it could be combined with `flow-events-mixin`? Traced back to [here](https://github.com/mozilla/fxa-content-server/issues/3608) but not sure if it still makes sense.